draft-ietf-quic-tls.md

```diff
@@ -1249,7 +1249,11 @@ quic_transport_parameters extension carries a TransportParameters struct when
 the version of QUIC defined in {{QUIC-TRANSPORT}} is used.
 
 The quic_transport_parameters extension is carried in the ClientHello and the
-EncryptedExtensions messages during the handshake.
+EncryptedExtensions messages during the handshake. Endpoints MUST send the
+quic_transport_parameters extension; endpoints that receive ClientHello or
+EncryptedExtensions messages without the quic_transport_parameters extension
+MUST terminate the TLS handshake with a fatal missing_extension alert (an error
```

Comment:
```suggestion
MUST close the connection with an error of type 0x16d (equivalent to a fatal TLS missing_extension alert, see Section {{tls-errors}}).
```

Reply:
This will result in "Section Section X.Y" in the final output.

Reply:
Also, that comma should be a semicolon; see #2626.